react/BaseContent.tsx

```diff
@@ -67,6 +67,7 @@ const Content: FC<Props> = ({ finishShoppingButtonLink }) => {
       </div>
       {!isCartEmpty && (
         <div className={minicartFooterClasses}>
+          {children}
```

Comment:
Make it more flexible verifying if children is passed and delegate the layout to the user.